src/drivers/driver_zephyr.c

```diff
@@ -1613,6 +1613,7 @@ int wpa_drv_zep_deinit_ap(void *priv)
 	}
 
 out:
+	if_ctx->beacon_set = false;
```

Comment:
Should this be protected with `if (if_ctx) {` (since this could crash for the `if (!priv) {` conditional code